src/pages/Task.tsx

```diff
@@ -127,6 +119,10 @@ const TaskCore: React.FC = () => {
   const logLinks = get(task, "logs");
   const isPerfPluginEnabled = false;
   const patchAuthor = data?.task.patchMetadata.author;
+  const failedTask =
+    task?.status?.includes(TaskStatus.Failed) ||
+    task?.status?.includes(TaskStatus.TaskTimedOut) ||
+    task?.status?.includes(TaskStatus.TestTimedOut);
 
```

Comment:
I dont think using  `.includes()` is necessary. Its better to just do a direct comparison. Using the .includes() will search the string for the result which is unnecessarily expensive. 
`task.status === TaskStatus.Failed || ....`